docs/core/extensions/custom-configuration-provider.md

```diff
@@ -33,6 +34,8 @@ Add an `EntityConfigurationContext` to store and access the configured values.
 
 :::code language="csharp" source="snippets/configuration/custom-provider/Providers/EntityConfigurationContext.cs":::
 
+By overriding <xref:Microsoft.EntityFrameworkCore.DbContext.OnConfiguring(Microsoft.EntityFrameworkCore.DbContextOptionsBuilder)> you can deterministically use the appropriate database connection. For example, if a connection string was provided you could connect to SQL Server, otherwise you could rely on an in-memory database.
```

Comment:
```suggestion
By overriding <xref:Microsoft.EntityFrameworkCore.DbContext.OnConfiguring(Microsoft.EntityFrameworkCore.DbContextOptionsBuilder)> you can use the appropriate database connection. For example, if a connection string was provided you could connect to SQL Server, otherwise you could rely on an in-memory database.
```